### DISCUSSION TITLE
OsmAnd for Paragliding rescue - FFVL tracker

### DISCUSSION THREAD

**Discussion by pierre-quelin** [osmandapp/OsmAnd#19332] [Ideas]
Hello,

first of all, thank you very much for your magnificent software, which I
wouldn't exchange for anything in the world.

I'm French and a paraglider. Every year, paragliders lose their lives
because nobody knows where to find and rescue them.

The "Fédération Française de Vol Libre" (FFVL) has set up a website and
an API to communicate our position during flights, saving precious time
in the event of a crash.

The FFVL site : https://federation.ffvl.fr/

Is there a plugin, even a very basic one, that could communicate our
current position using the API provided by the FFVL?

Thanks in advance

Pierre Quélin

**Comment by vshcherb** [comment]
I think we have Online Saving GPX track where you could specify custom URL, so probably it should work automatically. Please give a try.

**Comment by DmitryAlexei** [comment]
You can try Online tracking option in Trip recording plugin. For more information see https://osmand.net/docs/user/plugins/trip-recording#recording-settings

**Comment by pierre-quelin** [comment]
Thanks, I'll take a look and get back to you.

**Comment by pierre-quelin** [comment]
Hello, first of all thank you very much.

Technically the plugin as it stands does all that is necessary.
It could be improved by adding the following fields:
- Vertical speed in m/s
- Horizontal speed in m/s
- Location accuracy in meters.
- Heading of the device in degrees without decimal

The only remaining problem is on the server side, where it is necessary to send a "secret" application identifier. Entering it manually for each user poses a problem.

**Comment by pierre-quelin** [comment]
Hello, first of all thank you very much.

Technically the plugin as it stands does all that is necessary.
It could be improved by adding the following fields:
- Vertical speed in m/s
- Horizontal speed in m/s
- Location accuracy in meters.
- Heading of the device in degrees without decimal

The only remaining problem is on the server side, where it is necessary to send a "secret" application identifier. Entering it manually for each user poses a problem.

**Reply by vshcherb** [reply]
Well it's possible that some settings could be imported or url will be posted. We can add 
- Horizontal speed (I thought we had it)
- Location accuracy
- Heading of the device we can round without decimal 

So we will create a task for that

Vertical speed is not available.

**Reply by vshcherb** [reply]
I think accuracy available - https://docs.osmand.net/docs/user/plugins/trip-recording#distance-start-stop

It is real-time monitoring of your current location. If the option is enabled and the recording is in progress, the Distance/Start-Stop (REC) widget turns green instead of red, and each point is transmitted to a specified URL. The Web address field is for entering the URL using the parameter syntax: lat={0}, lon={1}, timestamp={2}, hdop={3}, altitude={4}, speed={5}, bearing={6}, eta={7} (unix time - to arrival), etfa={8} (unix time - to first intermediate point or finish point), eda={9} (distance in meters - to arrival or to a marker), edfa={10} (distance in meters - to first intermediate point or finish point). The Tracking Interval parameter is used to determine the frequency of sending location points. The options can be selected from 0 seconds to 5 minutes. The Time Buffer parameter sets the period of time that location points are saved if there is no Internet connection.

**Reply by pierre-quelin** [reply]
Thank you
